hms-app/src/components/NewIdea.tsx

```diff
@@ -21,85 +48,95 @@ function NewIdea() {
 
   function submitForm(event: any) {
```

Comment:
try to avoid any type, as it is basically avoiding type safety. Better go with:
`function submitForm(event: React.MouseEvent<HTMLButtonElement>) {`

Reply:
done